document-store/src/main/java/org/hypertrace/core/documentstore/model/options/DataFreshness.java

```diff
@@ -0,0 +1,9 @@
+package org.hypertrace.core.documentstore.model.options;
+
+@SuppressWarnings("UnnecessarySemicolon")
+public enum DataFreshness {
+  SYSTEM_DEFAULT,
+  REALTIME_FRESHNESS,
```

Comment:
Q: What does the freshness mean?

Reply:
Freshness means the liveliness of the data.
REALTIME_FRESHNESS means we want the most up-to-date data.
NEAR_REALTIME_FRESHNESS means a little bit of stale data is acceptable.

Reply:
Here `NEAR_REALTIME_FRESHNESS` means fetching data from secondary, right?

Reply:
Yes. Basically "SECONDARY" is preferred. In case "SECONDARY" is not available, it'll fetch from "PRIMARY".